WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -291,20 +286,28 @@ class AztecPostViewController: UIViewController {
         notificationCenter.removeObserver(self, name: .UIKeyboardWillHide, object: nil)
     }
 
-    func refreshNavigationBarButtons() {
-        refreshBlogPicker()
-        resizeBlogPicker()
+    func refreshInterface() {
+        reloadBlogPickerTitle()
+        reloadEditorContents()
+        resizeBlogPickerTitle()
     }
 
-    func refreshBlogPicker() {
+    func reloadEditorContents() {
+        let content = post.content ?? String()
+
+        titleTextField.text = post.postTitle
+        richTextView.setHTML(content)
+    }
+
+    func reloadBlogPickerTitle() {
         let blogName = post.blog.settings?.name ?? post.blog.url ?? String()
         let titleText = NSAttributedString(string: blogName, attributes: Constants.blogPickerAttributes)
 
         blogPickerButton.buttonMode = isSingleSiteMode ? .singleSite : .multipleSite
         blogPickerButton.setAttributedTitle(titleText, for: .normal)
     }
 
-    func resizeBlogPicker() {
+    func resizeBlogPickerTitle() {
```

Comment:
It's supposed to be working that way, but i didn't implement the `isEmpty` scenario. Thanks for catching this one!!